graylog2-server/src/main/java/org/graylog2/rest/resources/system/logs/LoggersResource.java

```diff
@@ -26,24 +26,19 @@
 import com.wordnik.swagger.annotations.ApiResponses;
 import org.apache.log4j.Level;
 import org.apache.log4j.Logger;
+import org.apache.log4j.spi.LoggingEvent;
 import org.apache.shiro.authz.annotation.RequiresAuthentication;
-import org.graylog2.rest.models.system.loggers.responses.LoggersSummary;
-import org.graylog2.rest.models.system.loggers.responses.SingleLoggerSummary;
-import org.graylog2.rest.models.system.loggers.responses.SingleSubsystemSummary;
-import org.graylog2.rest.models.system.loggers.responses.SubsystemSummary;
+import org.graylog2.MemoryAppender;
+import org.graylog2.rest.models.system.loggers.responses.*;
 import org.graylog2.shared.rest.resources.RestResource;
 import org.graylog2.shared.security.RestPermissions;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
 import org.slf4j.LoggerFactory;
 
-import javax.ws.rs.GET;
-import javax.ws.rs.NotFoundException;
-import javax.ws.rs.PUT;
-import javax.ws.rs.Path;
-import javax.ws.rs.PathParam;
-import javax.ws.rs.Produces;
+import javax.ws.rs.*;
 import javax.ws.rs.core.MediaType;
-import java.util.Enumeration;
-import java.util.Map;
+import java.util.*;
```

Comment:
We're not using wildcard imports at all in the Graylog codebase (as part of our informal coding standards). See http://stackoverflow.com/a/3348855/49505 for how to disable that in IntelliJ IDEA.